jinahub/segmenters/VADSpeechSegmenter/README.md

```diff
@@ -0,0 +1,90 @@
+# VADSpeechSegmenter
+
+**VADSpeechSegmenter** segments speech audio using Silero VAD (voice activity detector).
+
+
+Instead of segmenting according to time at fixed-length intervals, this segmenter segments based on voice activity.
+
+
+This VAD is similar to WebRTC but instead of differentiating voice from silence, this segmenter differenciates voice from noise/music/silence.
+
+
+**Table of Contents**
+
+- [🌱 Prerequisites](#-prerequisites)
+- [🚀 Usages](#-usages)
+- [🎉️ Example](#-example)
+- [🔍️ Reference](#-reference)
+
+
+## 🌱 Prerequisites
+
+> These are only needed if you download the source code and directly use the class. Not needed if you use the Jina Hub method below.
+
+In case you want to install the dependencies locally run
```

Comment:
`libsndfile1` and `ffmpeg` are also required, right?

Reply:
Yes, I have added them [here](https://github.com/jina-ai/executors/pull/97/commits/d6a5445f807522ab569712f843bc276be3b46a34), thanks.